hack/tools/release/notes.go

```diff
@@ -501,6 +501,10 @@ func generateReleaseNoteEntry(c *commit) (*releaseNoteEntry, error) {
 	}
 
 	if *prefixAreaLabel {
+		position := strings.Index(entry.title, area+":")
+		if position != -1 {
+			entry.title = strings.TrimSpace(entry.title[position+len(area+":"):])
+		}
```

Comment:
Nit:
what do you think about moving this to a separate method and do something here like
```go
entry.title = trimAreaFromTitle(title, area)
```

That way the code serves as "documentation" on what this piece of code does and why we want it. You can even add a go doc comment to the method.

Reply:
Yes ! I will do it